app/views/LocationShare/LocationPreviewModal.tsx

```
@@ -0,0 +1,178 @@
+		} catch (error) {
+			Alert.alert('Error', 'Could not open maps application');
+		}
+	};
+
+	const onCancel = () => Navigation.back();
+
+	const onShare = async () => {
+		try {
+			safeSet(() => setSubmitting(true));
+
+			const { url } = staticMapUrl(
```

ESLint

[error] 'url' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)